### ISSUE TITLE
about the loss

### ISSUE THREAD

**Issue by gravitychen** [yulu0724/EmbeddingDistillation#3]
In the training code , 
the loss was calculated like this
```
    if args.Ttype == 'relative':
            loss_dis[0].append(torch.mean(torch.norm(dis-dis_T,p=2)).data[0])
            loss_dis[1].append(0.0)
            loss_dis[2].append(0.0)
            
            loss_distillation = 0 * torch.mean(F.pairwise_distance(embed_feat,embed_feat_T))  # ~~~~~~~~~~ why we set the scalar to 0?
            loss_distillation += torch.mean(torch.norm(dis-dis_T,p=2))
            loss = loss_net + lamda * loss_distillation
```

why we set the scalar of the first loss_distillation definition to 0?

**Comment by yulu0724** [comment]
*0 is used to  initialized the distillation loss.

**Comment by gravitychen** [comment]
> *0 is used to initialized the distillation loss.

but if we use 0 as the distillation loss parameter, then does that means we just train it using metric loss only?

**Comment by gravitychen** [comment]
> *0 is used to initialized the distillation loss.

does it means that  distillation loss contribute nothing to the network updating?

**Comment by yulu0724** [comment]
Hi, *0 is used to initialized the distillation loss, and then  loss_distillation is computed by 'loss_distillation += torch.mean(torch.norm(dis-dis_T,p=2))'.
